server/graphql_schemas/utils/hash.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
Good job on utility class @tomipaul but what do you think if  `hash.py` is `hasher.py` since the class name is Hasher?